app/src/Controller/HumanAuthenticationsController.php

```diff
@@ -89,12 +89,14 @@ function ($recipient) use ($wblistRepository, $sender) {
             $form->get('email')->setData($senderEmail);
         }
 
+
         return $this->render('human_authentications/show.html.twig', [
-            'token' => $token,
-            'mailToValidate' => $senderEmail,
-            'form' => $form,
-            'confirm' => $confirm,
-            'content' => $content
+            'token'     => $token,
+            'mailToValidate'    => $senderEmail,
+            'form'      => $form,
+            'confirm'   => $confirm,
+            'content'   => $content,
+            'domain'    => $domain
```

Comment:
J’aime pas trop les modifs stylistiques de ce genre. Je vois l’intérêt d’un point de vue visuel, mais on finit par passer notre temps à changer le nombre d’espaces et ça ajoute du bruit à la relecture. Menfin c’est fait et c’est pas très grave non plus. Par contre j’ajoute toujours une virgule à la dernière ligne, ça permet d’avoir des diffs plus sympas quand on ajoute des lignes plus tard :


```suggestion
            'token'     => $token,
            'mailToValidate'    => $senderEmail,
            'form'      => $form,
            'confirm'   => $confirm,
            'content'   => $content,
            'domain'    => $domain,
```

Reply:
✅